src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -18,40 +26,72 @@ interface RelayProps extends React.HTMLProps<HTMLAnchorElement>, Props {
   }
 }
 
+@track({}, { dispatch: data => Events.postEvent(data) })
 class ArtistSearchResultsContent extends React.Component<RelayProps, null> {
   private excludedArtistIds: Set<string>
   followCount: number = 0
 
   constructor(props: RelayProps, context: any) {
     super(props, context)
-    this.excludedArtistIds = new Set(this.props.viewer.match_artist.map(item => item._id))
+    this.excludedArtistIds = new Set(
+      this.props.viewer.match_artist.map(item => item._id)
+    )
   }
 
-  onArtistFollowed(artistId: string, store: RecordSourceSelectorProxy, data: SelectorData): void {
-    const suggestedArtistEdge = data.followArtist.artist.related.suggested.edges[0]
+  onArtistFollowed(
+    artist: any,
```

Comment:
One change to note is that I'm passing in artist now instead of just `artist.__id` since we need `_id`, `id`, and `__id` in this function. This happens for all the subsequent files as well. Let me know if you see any problems with this!